rails/react-components/src/library/components/permission-forms-v2/students-tab/students-table.tsx

```diff
@@ -127,21 +128,32 @@ export const StudentsTable = ({ classId }: IProps) => {
   const handleSaveStudentPermissionsSuccess = async () => {
     setEditStudent(null);
     refetchStudentsData();
-  }
+  };
+
+  const handleClickPermissionExpandToggle = () => {
+    setPermissionsExpanded(!permissionsExpanded);
```

Comment:
When the new state is derived from the old one, I think we should theoretically use the `setPermissionsExpanded(oldValue => !oldValue);` form. While it's unlikely that users will click on this header multiple times, it's probably a good practice in general.